src/Resources/app/storefront/src/scss/base.scss

```diff
@@ -20,6 +20,8 @@
  *
  */
 
```

Comment:
This is due to [changes in how scss files are now loaded](https://github.com/shopware/platform/blob/6.4.0.0/UPGRADE-6.4.md#changed-the-loading-of-storefront-scss-files-in-extensions)